modules/loki-rbac-rules-permissions.adoc

```diff
@@ -0,0 +1,60 @@
+
+|`alertingrules.loki.grafana.com-v1-admin`
+|Users with this role have administrative-level access to manage alerting rules across the entire cluster. This cluster role grants permissions to create, read, update, delete, list, and watch `AlertingRule` resources within the `loki.grafana.com/v1` API group across all namespaces in the cluster.
+
+|`alertingrules.loki.grafana.com-v1-crdview`
+|Users with this role can view the definitions of Custom Resource Definitions (CRDs) related to `AlertingRule` resources within the `loki.grafana.com/v1` API group, but do not have permissions for modifying or managing these resources.
+
+|`alertingrules.loki.grafana.com-v1-edit`
+|Users with this role have permission to create, read, update, and delete `AlertingRule` resources. Users with this role can only manage `AlertingRule` resources within specified namespace(s).
+
+|`alertingrules.loki.grafana.com-v1-view`
+|Users with this role can read `AlertingRule` resources within the `loki.grafana.com/v1` API group. They can inspect configurations, labels, and annotations for existing alerting rules but cannot make any modifications to them. Users with this role can only read `AlertingRule` resources within specified namespace(s).
+
+|`recordingrules.loki.grafana.com-v1-admin`
+|Users with this role have administrative-level access to manage recording rules across the entire cluster. This cluster role grants permissions to create, read, update, delete, list, and watch `RecordingRule` resources within the `loki.grafana.com/v1` API group across all namespaces in the cluster.
+
+|`recordingrules.loki.grafana.com-v1-crdview`
+|Users with this role can view the definitions of Custom Resource Definitions (CRDs) related to `RecordingRule` resources within the `loki.grafana.com/v1` API group, but do not have permissions for modifying or managing these resources.
+
+|`recordingrules.loki.grafana.com-v1-edit`
+|Users with this role have permission to create, read, update, and delete `RecordingRule` resources. Users with this role can only manage `RecordingRule` resources within specified namespace(s).
+
+|`recordingrules.loki.grafana.com-v1-view`
+|Users with this role can read `RecordingRule` resources within the `loki.grafana.com/v1` API group. They can inspect configurations, labels, and annotations for existing alerting rules but cannot make any modifications to them. Users with this role can only read `RecordingRule` resources within specified namespace(s).
+
+|===
+
+To apply cluster roles for a user, you must bind an existing cluster role to a specific username.
+
+The following example command gives the specified user create, read, update and delete (CRUD) permissions for alerting rules in a specific namespace in the cluster:
+
+.Example cluster role binding command for alerting rule CRUD permissions in a specific namespace
+[source,terminal]
+----
+$ oc adm policy add-role-to-user alertingrules.loki.grafana.com-v1-edit -n <namespace> <username>
```

Comment:
I'd probably do the example with the `-admin` role, because that one allows read, write and delete operations.

```suggestion
$ oc adm policy add-role-to-user alertingrules.loki.grafana.com-v1-admin -n <namespace> <username>
```

Reply:
Got it, thank you!